packages/astro/test/content-collections-render.nodetest.js

```diff
@@ -1,10 +1,9 @@
-import { expect } from 'chai';
+import * as assert from 'node:assert/strict';
+import { after, describe, before, it } from 'node:test';
 import * as cheerio from 'cheerio';
-import { loadFixture, isWindows } from './test-utils.js';
+import { loadFixture } from './test-utils.js';
 import testAdapter from './test-adapter.js';
 
-const describe = isWindows ? global.describe.skip : global.describe;
```

Comment:
this line gave console error during local testing, so I have removed it

![image](https://github.com/withastro/astro/assets/88548999/6df8bac9-888a-42f8-aa46-632c476f88dc)

Reply:
What you can do is:

```js
if (!isWindows) {
	describe()
}
```